src/components/common/Extensions/UserHub/consts.ts

```diff
@@ -1,14 +1,15 @@
-import { CoinVertical, Receipt, User } from '@phosphor-icons/react';
+import { CoinVertical, Receipt } from '@phosphor-icons/react';
 import { defineMessages } from 'react-intl';
 
+import Balance from '~images/icons/balance.svg?react';
```

Comment:
This should be using the `Invoice` icon from `@phosphor-icons/react`

Reply:
Ohh that's pretty neat! I'll take note of that moving forward :) Thank you!

Reply:
Hey @iamsamgibbs that lib doesn't seem to have an export for `Invoice` , I'll give you a nudge on Discord in a few so you can walk me through that process if you don't mind 👀

Reply:
@iamsamgibbs bumped the lib and importing it now :) Cheers!